sdk/storage/Azure.Storage.Common/swagger/Generator/src/generator.ts

```diff
@@ -172,8 +174,9 @@ function generateOperation(w: IndentWriter, serviceModel: IServiceModel, group:
             w.write(`bool async = true`);
         }
         if (separateParams()) {  w.line(`,`); }
+        w.line(`bool ${bufferResponseName} = true,`);

```

Comment:
What do you think about adding a swagger attribute to mark endpoints that need buffering disabled or need an option to disable it?

Reply:
I was originally thinking of putting a swagger attribute to disable buffering but decided it'd be safer to let the user choose when calling (because if you're not wrapping in a RetriableStream, you should let us buffer your download).  I'm guessing you're worried about the 50 parameters on a lot of these methods?  😄  We can add an attribute to only expose the option on select APIs -- I'll do that as a separate change.